content/websocket-sample.md

```diff
@@ -0,0 +1,190 @@
++++
+title = "Fission WebSocket Sample"
+date = "2021-06-11T10:50:51+05:30"
+author = "Gaurav Gahlot"
+description = "Write a serverless broadcast function with NodeJS"
+categories = ["Fission", "Serverless", "Application"]
+type = "post"
++++
+
+## Introduction
+
+In this post we will be into how we can develop a simple chat application using Fission functions.
+Fission's NodeJS environment now has built in support for [WebSocket][1].
+So, we are going to use this environment to power our simple web based chat application.
+
+Let's first understand how this is going to work.
+
+![architecture](../../../images/fission-websocket-sample.png)
+
+- We start by creating a NodeJS environment using the [fission/nodejs-env][2] Docker image.
+- Then we create a fission function for with [broadcast.js][3].
+It is this piece of code that will broadcast a message to all connected clients.
+- Next, we will create an [HTTP trigger][4] for the above created function.
+- We then update our chat application to connect over the route, we just created.
+- Start the HTTP server to serve the chat application and open two instances of the application in separate browser windows.
+- As you send a message from one chat window, the function will be triggered and the message will be broadcasted to all connected clients. 
+
+We will cover two approaches using which you can test this setup.
+
+## Prerequisites
+
+You will need a Kubernetes cluster with Fission installed.
+Please follow the [instructions here][5] for installing Fission in your Kubernetes cluster.
+Then, verify that the fission cli is working by using the `fission --version` command.
```

Comment:
Maybe we should mention this will work from Fission 1.13

Reply:
@gauravgahlot This is not resolved yet.